tower-http/src/body.rs

```diff
@@ -0,0 +1,121 @@
+//! Body types.
+//!
+//! All these are wrappers around other body types. You shouldn't have to use them in your code.
+//! Use `http-body-util` instead.
+//!
+//! They exist because we don't want to expose types from `http-body-util` in `tower-http`s public
```

Comment:
Why not?

Reply:
The futures crates, including `futures-util`, are very stable. `http-body-util` is much more likely to see breaking-change releases soon-ish, and if we had it as a public dependency, it would require a breaking-change release of `tower-http` to upgrade to the latest version.

Reply:
Understood. Something I am less concerned about. But I understand that this is for tower not an option.